src/AvaloniaEdit/Rendering/CollapsedLineSection.cs

```diff
@@ -27,82 +27,76 @@ namespace AvaloniaEdit.Rendering
 	/// </summary>
 	public sealed class CollapsedLineSection
 	{
-	    private DocumentLine _start;
-	    private DocumentLine _end;
-	    private readonly HeightTree _heightTree;
-		
-		#if DEBUG
+		private readonly HeightTree _heightTree;
+
+#if DEBUG
 		internal string Id;
-	    private static int _nextId;
-		#else
-		internal const string Id = "";
-		#endif
-		
+		private static int _nextId;
+#else
+		const string ID = "";
```

Comment:
```suggestion
		const string Id = "";
```

Otherwise, it will break compilation in Release config.